website/project/views/node.py

```diff
@@ -732,6 +731,9 @@ def _view_project(node, auth, primary=False):
             'tags': [tag._primary_key for tag in node.tags],
             'children': bool(node.nodes),
             'is_registration': node.is_registration,
+            'is_retracted': getattr(node.retraction, 'is_retracted', False),
```

Comment:
It might make sense to add `Node.is_retracted`, `Node.pending_retraction`, etc. properties.